web/pdf_viewer.js

```diff
@@ -2350,7 +2350,8 @@ class PDFViewer {
       const idsToRefresh = this.#switchToEditAnnotationMode();
       if (isEditing && idsToRefresh) {
         // We're editing so we must switch to editing mode when the rendering is
-        // done.
+        // done and the structure tree parsed (in order to use some structure
+        // tree data in some existing editors).
```

Comment:
Let's revert the comment now that the code changes are removed.
```suggestion
        // done.
```